bin/strata-client/src/rpc_server.rs

```diff
@@ -660,6 +701,21 @@ impl SequencerServerImpl {
 
 #[async_trait]
 impl StrataSequencerApiServer for SequencerServerImpl {
+    async fn get_last_broadcast_entry(&self) -> RpcResult<Option<L1TxEntry>> {
+        let broadcast_handle: Arc<L1BroadcastHandle> = self.broadcast_handle.clone();
+        let txentry = broadcast_handle.get_last_broadcast_entry().await;
+        Ok(txentry.map_err(|e| Error::Other(e.to_string()))?)
+    }
+
+    async fn get_broadcast_entry_by_idx(&self, txid: HexBytes32) -> RpcResult<Option<L1TxEntry>> {
+        let mut txid = txid.0;
+        txid.reverse();
+        let id = Buf32::from(txid);
```

Comment:
We shouldn't be doing this parsing all on the fly, should have some better way to deal with it.

Also this says "idx" in the fn name but it's taking a txid, what's the deal with that?

Reply:
Thanks! It should be by idx.